public/CSS/style.css

```
@@ -0,0 +1,161 @@
+/** reset **/
+* {
+    margin: 0;
+    padding: 0;
+  }
+
+  a {
+    text-decoration: none;
+    color: inherit;
+  }
+  
+  /* True Styling */
+  
+  header {
+    background-image: url(https://actualitte.com/media/cache/width_944_webp/uploads/images/un-nouveau-projet-d-hayao-miyazaki-inspire-d-un-classique-de-la-litterature-japonaise-6950fb53c799.jpg);
+    background-size: cover; 
+    background-repeat: no-repeat;
+    display: flex;
+    flex-direction: column;
+  }
+  
+  header h1 {
+    margin: 0;
+    font-family: fantasy;
+    font-size: 5rem;
+    padding: 10px;
+    text-align: center;
+    color: white!important;
+
+    text-shadow: 0px 0px 10px rgba(0,0,0,0.5), 0px 0px 10px rgba(0,0,0,0.5), 0px 0px 10px rgba(0,0,0,0.5);
+  }
+  
+  header nav{
+    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+    flex-direction: column;
+  }
+  
+  header nav ul {
+    background: rgba(255, 255, 255, 0.486);
+    color: gb(243, 205, 18);
```

```suggestion
    color: rgb(243, 205, 18);
```

de manière général, j'ai tendance à utiliser plutôt les `#HEXADECIMAL` mais c'est une plus une question de préférence je crois ^^

---

Tu peux tester https://stylelint.io/ pour avoir un linter côté CSS